Hunted Mobile/Hunted Mobile/Hunted Mobile/Service/Json/EventJsonService.cs

```diff
@@ -45,4 +47,22 @@ public class ScoreUpdatedEventJsonService : JsonConversionService<ScoreUpdatedEv
             };
         }
     }
+
+    public class GadgetsUpdatedEventJsonService : JsonConversionService<GadgetsUpdatedEventData, Model.Response.Json.GadgetsUpdatedEventData> {
+        public override GadgetsUpdatedEventData ToObject(Model.Response.Json.GadgetsUpdatedEventData data) {
+            var gadgets = new GadgetJsonService().ToObjects(data.gadgets);
+            var flattenedGadgets = new List<Gadget>();
+            foreach(var gadgetsOfType in gadgets) {
+                foreach(var gadgetOfType in gadgetsOfType) {
```

Comment:
Wat gebeurt hier precies? Die naamgeving maakt het een beetje onduidelijk. Bedoel je niet gewoon `foreach(var gadget in gadgets) {` en daarna `foreach(var gadgetOfType in gadget) {`. Of `type in gadget`

Reply:
Deze variabelnamen zijn niet heel duidelijk idd, maar omdat gadgets in de database een amount hebben maak ik voor elk amount een object aan en voeg alle gadgets van verschillende types samen in 1 lijst (als dat nog te volgen is).